go.mod

```diff
@@ -81,7 +81,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	go.viam.com/api v0.1.467
+	go.viam.com/api v0.1.472
```

Comment:
I think the version that's actually needed is 471. Can we just change it to that so we don't accidentally pull in other changes?